src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/VisualRuleForm.types.ts

```diff
@@ -0,0 +1,38 @@
+import {
+    IButtonStyles,
+    IDropdownStyles,
+    ILabelStyles,
+    IStyleFunctionOrObject,
+    ITextFieldStyles,
+    Theme
+} from '@fluentui/react';
+import { ITooltipCalloutStyles } from '../../../TooltipCallout/TooltipCallout.types';
+import { IConditionsListStyles } from './Internal/ConditionsList.types';
+
+export interface IVisualRuleFormProps {
+    isPropertyTypeDropdownEnabled: boolean;
```

Comment:
please see my other comments about instead of passing both isPropertyTypeDropdownEnabled and setPropertyTypeDropdownEnabled, making that internal to this component.